internal/cloudapi/v2/openapi.v2.yml

```diff
@@ -792,6 +793,13 @@ components:
       properties:
         repo_url:
           type: string
+    LocalUploadStatus:
+      type: object
+      required:
+        - filename
+      properties:
+        filename:
+          type: string
```

Comment:
This is not an actual filename but AFAICT, an absolute path to the artifact. I'd suggest renaming it to something that expresses that fact.